src/levenshtein.cr

```diff
@@ -3,10 +3,12 @@ module Levenshtein
   # Computes the [levenshtein distance](http://en.wikipedia.org/wiki/Levenshtein_distance) of two strings.
   #
   # ```
-  # levenshtein("algorithm", "altruistic") # => 6
-  # levenshtein("hello", "hallo")          # => 1
-  # levenshtein("こんにちは", "こんちは")           # => 1
-  # levensthein("hey", "hey")              # => 0
+  # require "levenshtein"
+  #
+  # Levenshtein.distance("algorithm", "altruistic") # => 6
+  # Levenshtein.distance("hello", "hallo")          # => 1
+  # Levenshtein.distance("こんにちは", "こんちは")           # => 1
```

Comment:
@Sija I'm pretty sure this might end up being screwy no matter what; the Unicode symbols used are variable-width, so different fonts are all going to show it a bit differently.